tensorflow/core/kernels/depthtospace_op.cc

```diff
@@ -172,6 +172,16 @@ struct DepthToSpaceOpFunctor<CPUDevice, T, FORMAT_NHWC> {
     }
   }
 };
+
+#ifdef WIN32
+template <typename T>
+struct DepthToSpaceOpFunctor<CPUDevice, T, FORMAT_NCHW> {
+  void operator()(const CPUDevice& d, typename TTypes<T, 4>::ConstTensor input,
+                  int block_size, typename TTypes<T, 4>::Tensor output) {
+    LOG(FATAL) << "dummy implementation to make debug build compile";
+  }
+};
+#endif
```

Comment:
Do you have some intuition for why these are needed only on Windows debug build?

Reply:
The `if (std::is_same<Device, GPUDevice>::value) {` condition in `DepthToSpaceOp::Compute()` obviously never becomes true for CPU devices. In release mode, compilers optimize out this whole block for CPU device template instantiations. In debug mode, non-Windows compilers seem to "understand" that this branch is dead, too, MSVC not...